[08/Sep/2014 21:01:07] ERROR [django.request:231] Internal Server Error: /admin/core/notice/add/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 567, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 105, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/decorators/cache.py", line 52, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 204, in inner
    return view(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 105, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/transaction.py", line 394, in inner
    return func(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1388, in changeform_view
    self.save_model(request, new_object, form, not add)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1029, in save_model
    obj.save()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 284, in save
    super(Notice, self).save(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/base.py", line 590, in save
    force_update=force_update, update_fields=update_fields)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/base.py", line 618, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/base.py", line 699, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/base.py", line 732, in _do_insert
    using=using, raw=raw)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/query.py", line 921, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 920, in execute_sql
    cursor.execute(sql, params)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/backends/utils.py", line 81, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
IntegrityError: null value in column "photographer_id" violates not-null constraint
DETAIL:  Failing row contains (2, notice, 0, 0, 0, t, t, 2003-06-01, 2014-09-09 00:01:07.425139+00, Fugiat eos beatae accusamus iure dignissimos sunt blanditiis s, Molestiae ut eiusmod quia ullam suscipit est voluptates, <p>qweqwe</p>
, 1, 2, 2, 3, news/o1351169_X4bJVwT.jpg, null).

[08/Sep/2014 21:05:42] INFO [system.core.generic_view:171] Template loaded: gui/home.html
[08/Sep/2014 21:14:53] INFO [system.core.generic_view:171] Template loaded: gui/events.html
[08/Sep/2014 21:14:58] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 13:07:10] INFO [system.core.generic_view:171] Template loaded: gui/home.html
[10/Sep/2014 14:10:35] INFO [system.core.generic_view:171] Template loaded: gui/notice.html
[10/Sep/2014 14:10:43] INFO [system.core.generic_view:171] Template loaded: gui/events.html
[10/Sep/2014 14:10:47] ERROR [system.core.views:123] "'editorial'"
[10/Sep/2014 14:10:47] ERROR [system.core.views:140] "'keywords'"
[10/Sep/2014 14:10:47] INFO [system.core.generic_view:171] Template loaded: gui/posts.html
[10/Sep/2014 14:10:59] ERROR [system.core.views:123] "'editorial'"
[10/Sep/2014 14:10:59] ERROR [system.core.views:140] "'keywords'"
[10/Sep/2014 14:10:59] INFO [system.core.generic_view:171] Template loaded: gui/posts.html
[10/Sep/2014 14:11:05] INFO [system.core.generic_view:171] Template loaded: gui/events.html
[10/Sep/2014 14:11:06] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:16:09] WARNING [django.request:143] Not Found: /favicon.ico
[10/Sep/2014 14:23:04] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:26:33] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:26:33] ERROR [django.request:231] Internal Server Error: /event/2
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 437, in all_videos
    return [video_libraries.get_videos for video_library in self.video_libraries]
TypeError: 'ManyRelatedManager' object is not iterable
[10/Sep/2014 14:28:01] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:28:01] ERROR [django.request:231] Internal Server Error: /event/2
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 437, in all_videos
    return [video_libraries.get_videos for video_library in self.video_libraries.all]
TypeError: 'instancemethod' object is not iterable
[10/Sep/2014 14:28:19] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:28:19] ERROR [django.request:231] Internal Server Error: /event/2
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 437, in all_videos
    return [video_libraries.get_videos for video_library in self.video_libraries.all()]
NameError: global name 'video_libraries' is not defined
[10/Sep/2014 14:28:42] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:28:42] ERROR [django.request:231] Internal Server Error: /event/2
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 437, in all_videos
    return [video_library.get_videos for video_library in self.video_libraries.all()]
NameError: global name 'video_libraries' is not defined
[10/Sep/2014 14:32:36] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:32:37] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 14:33:16] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:24:56] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:25:35] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:25:35] ERROR [django.request:231] Internal Server Error: /event/2
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 437, in all_videos
    print [Video.objects.filter(self.video_library) for self.video_library in self.video_libraries.all()]
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/query.py", line 691, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/query.py", line 709, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1287, in add_q
    clause, require_inner = self._add_q(where_part, self.used_aliases)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1314, in _add_q
    current_negated=current_negated, connector=connector)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/models/sql/query.py", line 1135, in build_filter
    arg, value = filter_expr
TypeError: 'VideoLibrary' object is not iterable
[10/Sep/2014 19:26:57] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:27:54] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:28:08] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:28:27] INFO [system.core.generic_view:171] Template loaded: gui/events.html
[10/Sep/2014 19:28:47] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:29:05] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:29:30] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:29:59] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:30:10] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:30:50] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:31:24] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:31:24] ERROR [django.request:231] Internal Server Error: /event/3
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 439, in all_videos
    print video.video
AttributeError: 'QuerySet' object has no attribute 'video'
[10/Sep/2014 19:31:25] WARNING [django.request:143] Not Found: /favicon.ico
[10/Sep/2014 19:31:35] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:31:35] ERROR [django.request:231] Internal Server Error: /event/3
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 439, in all_videos
    print video
AttributeError: 'QuerySet' object has no attribute 'video'
[10/Sep/2014 19:31:45] INFO [system.core.generic_view:171] Template loaded: gui/event.html
[10/Sep/2014 19:31:45] ERROR [django.request:231] Internal Server Error: /event/3
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/fotec/system/core/generic_view.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts.py", line 23, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 178, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 80, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 155, in render
    values = self.sequence.resolve(context, True)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 596, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 734, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 788, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/fotec/system/core/models.py", line 439, in all_videos
    print video
AttributeError: 'QuerySet' object has no attribute 'video'
[10/Sep/2014 19:31:46] INFO [system.core.generic_view:171] Template loaded: gui/event.html
